<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NanHao_main_votes</class>
 <widget class="QDialog" name="NanHao_main_votes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NanHao_main_votes</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2" rowspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>23</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>502</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3" rowspan="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>172</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4" rowspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>38</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="6" rowspan="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3" colspan="3">
    <widget class="QPushButton" name="pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>华文新魏</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>设置扫描仪</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>65</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="3" colspan="3">
    <widget class="QPushButton" name="exitBt_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>华文新魏</family>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
   </item>
   <item row="9" column="6">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>26</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3" colspan="3">
    <widget class="QPushButton" name="pushButton_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>华文新魏</family>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>开始计票</string>
     </property>
    </widget>
   </item>
   <item row="5" column="6">
    <spacer name="verticalSpacer_8">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>11</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="6">
    <spacer name="verticalSpacer_9">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>11</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="3" colspan="3">
    <widget class="QPushButton" name="exitBt">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>华文新魏</family>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </item>
   <item row="8" column="6">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>65</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="6">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>65</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="6">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>65</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="6">
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>11</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="1">
    <spacer name="verticalSpacer_10">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" rowspan="7">
    <widget class="QTableView" name="listTv"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="NanHao.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exitBt</sender>
   <signal>clicked()</signal>
   <receiver>NanHao_main_votes</receiver>
   <slot>returns()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>675</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>784</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>NanHao_main_votes</receiver>
   <slot>scan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>679</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>788</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>NanHao_main_votes</receiver>
   <slot>setting_scanner()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>679</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>783</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitBt_2</sender>
   <signal>clicked()</signal>
   <receiver>NanHao_main_votes</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>698</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>787</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setting()</slot>
  <slot>clear()</slot>
  <slot>setting_scanner()</slot>
 </slots>
</ui>
